types/chain-information.type.ts

```diff
@@ -0,0 +1,21 @@
+export type ChainInformation = {
+  chainName: string
+  description: string
+  platformChainId: string
+  subnetId: string
+  vmId: string
+  vmName: string
+  rpcUrl: string // uri
+  wsUrl: string // uri
+  multicallAddress: string // ChecksumAddress
+  isTestnet: boolean
+  networkToken: {
```

Comment:
Should I get rid of this file completely or should I just copy exactly what is in the chain-sdk interfaces

Reply:
@dfenstermaker I would actually suggest that the `chain-sdk` uses whatever type is stored in here since this repo is the source of truth for the data shape.